gpuselect/__init__.py

```diff
@@ -23,10 +24,10 @@ def get_gpu(gpu_weight, mem_weight):
             time.sleep(.01)
         G.append(np.mean(gpuutil))
         M.append(np.mean(memutil))
-    print "GPU Utilization:", G
-    print "Mem Utilization:", M
-    print "GPU Weight     :", gpu_weight
-    print "Mem Weight     :", mem_weight
+    print ("GPU Utilization:", G)
```

Comment:
Because you're into flake8 and formatting, I'll mention that I think there should be no space before the bracket.